gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/KubernetesConfiguredControllerIT.java

```diff
@@ -45,4 +46,23 @@ void k8sResourceTask_whenRunWithConfiguredControllerType_generatesK8sManifestWit
         .contains("Adding revision history limit to 2")
         .contains("validating");
   }
+
+  @Test
+  void k8sResourceTask_whenRunWithCronJobController_generatesK8sManifestWithCronJobController() throws IOException, ParseException {
+    // When
```

Comment:
Umm, isn't it possible to achieve this inside parameterized test itself?

Maybe you can add property or groovy DSL config for schedule in controller project? That would only be applicable when controller type is CronJob.